### DISCUSSION TITLE
Zero value of control input in output log

### DISCUSSION THREAD

**Discussion by rishi1906** [JSBSim-Team/jsbsim#1098] [Q&A] [answered]
Hi,

In csv file I get zero value for all control inputs namely 
```
fcs/pitch-sum,
fcs/elevator-control,
fcs/elevator-actuator,
fcs/roll-sum,
fcs/left-aileron-control,
fcs/left-aileron-actuator,
fcs/right-aileron-control,
fcs/right-aileron-actuator,
fcs/effective-aileron-pos,
Rudder Command Sum, 
Rudder Control,
Rudder Normalized,
electric HP (engine 0),
Direct Thrust (engine 0 in lbs)
```
@seanmcleod, am I missing some tag to enable these values to be fetched in configuration file.

**Comment by seanmcleod** [comment]
Where are you getting the list above from?

**Comment by rishi1906** [comment]
Apologies for the late reply; I got caught up with work. I am using Ardupilot SITL for simulation. Following lines of code in xml configuration file outputs above mentioned data along with several other parameters in a csv file.

```xml
<output name="JSBout.csv" type="CSV" rate="10">
        <rates> ON             </rates>
        <velocities> ON        </velocities>
        <position> ON          </position>
        <atmosphere> ON       </atmosphere>
        <fcs> ON               </fcs>
        <ground_reactions> OFF </ground_reactions>
        <propulsion> ON       </propulsion>
        <simulation> ON        </simulation>
        <massprops> ON        </massprops>
        <forces> ON           </forces>
        <moments> ON          </moments>
        <aerosurfaces> ON      </aerosurfaces>
    </output>
```
I need to know thrust value of simulated flight for validation.

**Comment by rishi1906** [comment, marked as answer]
I am getting correct values in output log file now. Closing this discussion. 

![image](https://github.com/JSBSim-Team/jsbsim/assets/16606630/9e9d4c76-80ab-4022-a95b-d83ae8220bb7)
